extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1307,6 +1318,9 @@ export class AmpA4A extends AMP.BaseElement {
         /** @type {!Document} */ (this.element.ownerDocument),
         'iframe', /** @type {!JsonObject} */ (
         Object.assign(mergedAttributes, SHARED_IFRAME_PROPERTIES)));
+    if (this.isFluid && this.iframe) {
+      this.setupListenersForFluid(this.iframe);
```

Comment:
I'm assuming you did this because of the possible race condition of doing it within onCreativeRender given it executes after appendChild?  Hmmm... I'm slightly concerned with moving it to prior to attach but I'm also not sure it would have a negative effect.  Please investigate (and we should wrap in experiment flag so we can undo if any issues arise).

Reply:
No, the reason I put this here is because onCreativeRender is too late. At that point the iframe is attached and iframe-helper doesn't seem to want to register events to attached iframes. See https://github.com/ampproject/amphtml/blob/master/src/iframe-helper.js#L251.